services/provider/server/server.go

```diff
@@ -80,17 +82,32 @@ func NewOCSProviderServer(ctx context.Context, namespace string) (*OCSProviderSe
 		return nil, fmt.Errorf("failed to create new StorageClassRequest instance. %v", err)
 	}
 
+	version, err := semver.Make(ocsVersion.Version)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse operator version as semver. %v", err)
+	}
+
 	return &OCSProviderServer{
 		client:                     client,
 		consumerManager:            consumerManager,
 		storageClassRequestManager: storageClassRequestManager,
 		namespace:                  namespace,
+		version:                    &version,
```

Comment:
Why do we need to have this on the OCSProviderServer struct? It is always available at `ocsVersion.Version`

Reply:
> It is always available at ocsVersion.Version
- but not as semver.Version
- I'm also fine doing the conversion every time but that'll be redundant

Reply:
done.